include/seqan3/range/views/minimiser_hash.hpp

```diff
@@ -0,0 +1,156 @@
+     */
+    template <std::ranges::range urng_t>
+    constexpr auto operator()(urng_t && urange, shape const & shape, uint32_t const window_size,
+                              uint64_t const seed = 0x8F3F73B5CF1C9ADE) const
+    {
+        static_assert(std::ranges::viewable_range<urng_t>,
+            "The range parameter to views::minimiser_hash cannot be a temporary of a non-view range.");
+        static_assert(std::ranges::forward_range<urng_t>,
+            "The range parameter to views::minimiser_hash must model std::ranges::forward_range.");
+        static_assert(semialphabet<reference_t<urng_t>>,
+            "The range parameter to views::minimiser_hash must be over elements of seqan3::semialphabet.");
+
+        if (shape.size() > window_size)
+            throw std::invalid_argument{"The size of the shape cannot be greater than the window size."};
+
+        return std::forward<urng_t>(urange) | seqan3::views::kmer_hash(shape)
+                                            | std::views::transform([seed] (uint64_t i) { return i ^ seed; })
+                                            | seqan3::views::minimiser(window_size - shape.size() + 1);
+    }
+};
+
+} // namespace seqan3::detail
+
+namespace seqan3::views
+{
+
+/*!\name Alphabet related views
+ * \{
+ */
+
+/*!\brief                    Computes minimisers for a range with a given shape, window size and seed.
+ * \tparam urng_t            The type of the range being processed. See below for requirements. [template parameter is
+ *                           omitted in pipe notation]
+ * \param[in] urange         The range being processed. [parameter is omitted in pipe notation]
+ * \param[in] shape          The seqan3::shape that determines how to compute the hash value.
+ * \param[in] window_size    The window size to use.
+ * \param[in] seed           The seed to use. Default: 0x8F3F73B5CF1C9ADE.
+ * \returns                  A range of `size_t` where each value is the minimiser of the resp. window.
+ *                           See below for the properties of the returned range.
+ * \ingroup views
+ *
+ * \details
+ *
+ * A sequence can be presented by a small number of k-mers (minimisers). For a given shape and window size all k-mers
+ * are determined in the forward strand and only the lexicographically smallest k-mer is saved for
+ * one window. This process is repeated over every possible window of a sequence. If consecutive windows share a
+ * minimiser, it is saved only once. For example, in the sequence "TAAAGTGCTAAA" for an ungapped shape of length 3 and a
+ * window size of 5 the first, the second and the last window contain the same minimiser "AAA".
+ * Because the first two windows are consecutive and storing the minimiser "AAA" twice is redundant, it is stored only
+ * once. The "AAA" miniser of the last window on the other hand is stored, because the last window is not consecutive to
+ * the other two and storing the second "AAA"-minimiser is also not redundant.
```

Comment:
```suggestion
 * Because the minimisers of the first two consecutive windows also share the same position, storing this minimiser twice is redundant and it is stored only
 * once. The "AAA" minimiser of the last window on the other hand is stored, since it is located at an other position than the previous "AAA" minimiser and hence storing the second "AAA"-minimiser is not redundant but necessary.
```

+ some linebreaks

Reply:
@MitraDarja I also changed the text, the linebreak comment is just because I did not add the linebreaks in my suggestion

Reply:
Ups, sorry, I misunderstood! :sweat_smile:

Reply:
No problem, I wrote `+ some linebreaks` which renders to a bullet point, instead of `\+ some linebreaks` which would show the actual `+`